storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Test/BoltOutput.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Dotnet.Storm.Adapter.Messaging;
+using Dotnet.Storm.Adapter.Components;
+
+namespace Dotnet.Storm.Adapter.Test
+{
+    public class BoltOutput : TestOutput
+    {
+        public List<string> Anchors { get; set; }
+
+        internal BoltOutput(BoltTuple bt)
+        {
+            Anchors = bt.Anchors;
+            this.Stream = bt.Stream;
+            this.Task = bt.Task;
+            this.Tuple = bt.Tuple;
+            this.NeedTaskIds = bt.NeedTaskIds;
+            this.ComponentId = Component.Context.ComponentId;
```

Comment:
I do believe we don't need component id here.

Reply:
We need it in tests. Please refer to ComponentID of FakeStormContext.cs